src/components/item-details/bid-tab/history-tab-content.tsx

```diff
@@ -1,23 +1,63 @@
-import TopSeller from '@components/top-seller/top-seller-2';
-import { History } from '@types';
+import { History, Item } from '@types';
+import Anchor from '@ui/anchor';
+import { getTimeAgo } from '@utils/getTimeAgo';
+import clsx from 'clsx';
+import Image from 'next/image';
+import { Fragment } from 'react';
+import useSWR from 'swr';
+import { historyService } from '../../../features/marketplace/services/history-service';
 
 type Props = {
-  history: History[];
+  item: Item;
 };
+const fetchHistory = async (itemId: string) =>
+  historyService.findAllByItemId(itemId).then((data) => data.reverse());
 
-const HistoryTabContent = ({ history }: Props) => (
-  <div>
-    {history?.map((item: History) => (
-      <TopSeller
-        key={item.id}
-        name={item.user.name}
-        eth={item.amount}
-        path={item.user.slug}
-        time={item.bidAt}
-        image={item.user.image}
-      />
-    ))}
-  </div>
-);
+export const HistoryTabContent = ({ item }: Props) => {
+  const { data: history } = useSWR<History[]>(item.itemId, fetchHistory);
```

Comment:
Hey @mmiranda-stackit check this

Reply:
I won't argue about it, it is a different way to achieve the same thing (among other things). The only thing about it though, is that I would like to keep standard the way we are already fetching data (this is for keep the app consistent and not have to deal with two or more different approaches). It would be nice if we can explore other ways to do certain things, but decide where and when.

Reply:
Ohh don't worry, I just wanted show you this one